src/index.ts

```diff
@@ -53,7 +53,13 @@ function cleanUpTempStorage() {
 
 async function monitor(): Promise<void> {
   try {
-    logger.info({ cluster: currentClusterName }, 'starting to monitor');
+    logger.info(
+      {
+        cluster: currentClusterName,
+        useKeepalive: config.USE_KEEPALIVE,
```

Comment:
so this is just sanity checked to see if the `USE_KEEPALIVE` was set?

Reply:
sure 🤷